workflow/scripts/osemosys_global/summary/gen_shares.py

```diff
@@ -0,0 +1,78 @@
+"""Calcualtes Renewable Generation"""
+
+import pandas as pd
+from typing import Optional
+from constants import CLEAN, RENEWABLES, FOSSIL
+
+# do not include storage in generation share calcualtion 
+EXCLUDE = ["LDS", "SDS"]
```

Comment:
This should be read dynamically from the storage_parameters entry in the config file since the storage technologies can change.